katsdpimageutils/primarybeamcorrection/primary_beam_correction.py

```diff
@@ -191,8 +222,14 @@ def primary_beam_correction(beam_pattern, path):
 def _get_value_from_history(keyword, header):
     """
     Return the value of a keyword from the FITS HISTORY in header
-    Assumes keyword is found in a line of the HISTORY with format: 'keyword = value'
-    Input: Keyword [e.g BMAJ, CLEANBMJ, BMIN] and Image header
+    Assumes keyword is found in a line of the HISTORY with format: 'keyword = value'.
+    
+    Parameters
+    ----------
+      Keyword: str
+          keyword to search for such as BMAJ, CLEANBMJ and BMIN
+      header: astropy object
+          Image header
```

Comment:
The parameter names should have the same indentation as the `Parameters` section header.

Reply:
Done.